mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx

```diff
@@ -515,6 +519,39 @@ export function PortfolioScreen({ navigation }: Props): JSX.Element {
     },
   };
 
+  const resetNavigationStack = () => {
```

Comment:
I think this will fix also the issue when change languages. Can you add this fix also?

Reply:
We will address language switch reset on a separate PR

Reply:
https://github.com/DeFiCh/wallet/pull/3691